### ISSUE TITLE
THIS PROJECT IS LOOKING FOR A NEW OWNER

### ISSUE THREAD

**Issue by bradvogel** [bee-queue/bee-queue#528]
Hey everyone!

The [Mixmax Engineering team](https://mixmax.com/engineering) can no longer maintain this repo. We're moving away from Bee Queue over to SQS.

We're looking for a new owner of this repo.

You:
* are an experienced engineer working on a team that's using Bee Queue at scale.
* have extra time to be able to review and merge PRs.
* have an exceptional attention to quality. Many people and organizations rely on this project! Let's keep it stable.

If you're approved, we will:
* give you write access to this repo in order to merge PRs
* be available if you have any technical questions

**Comment by compwright** [comment]
![](https://media.giphy.com/media/fWx4030lcRWUtQ7H9T/giphy.gif)

I'm an experienced engineer using bee-queue in production. I'm not sure what qualifies as "at scale." I meet the other criteria and have a number of open source packages that I maintain.

**Comment by bradvogel** [comment]
@compwright Thanks!

Are there PRs in here that you feel are ready to merge? Please comment in them. I'll double check the code, then give you write access if we align.

**Comment by compwright** [comment]
@bradvogel yes, for starters all the dependabot PRs should be given priority.

There are a handful of others, and I will tag you in a comment in them.

**Comment by compwright** [comment]
@tmeasday you interested in helping out as a maintainer?

**Comment by compwright** [comment]
@bradvogel asked if I knew of anyone that could join the maintainers team, and you came to mind. Right now it's just me. Best for the project if there are long term maintainers besides me. But I'd take short term help while I get acclimated.

**Comment by tmeasday** [comment]
For sure, ping me when you need. I'm more familiar just as a consumer, the only code I wrote was 3 years ago and even at that stage I found the test strategy pretty hard to understand, but I get the basic code flow in the package.

**Comment by jorenvandeweyer** [comment]
We use bee queue in production ~400 jobs/s. I'm willing to review PR's.

**Comment by compwright** [comment]
Thank you @jorenvandeweyer!

**Comment by entrptaher** [comment]
I made a short experiment based on bee-queue with a better worker_threads this evening only to find this issue :D 

https://github.com/entrptaher/extended-bee-queue

**Comment by dogabudak** [comment]
We are using bee-queue in production (couple hundres thousand messages per day). I am willing to contribute.

**Comment by bradvogel** [comment]
@dogabudak thanks! Please send PRs our way and we'll review. Once trust is earned, we'll make you a maintainer.

**Comment by Rohithzr** [comment]
Is there someone working on the ioredis implementation or is that abandoned? if it's still something people want, I can work on adding that capability. If this is not something the community is looking for, then I will switch to a standalone Redis cluster for now.

**Comment by LeGrizzly** [comment]
Hello @bradvogel,

Thank you for your work on this project. I wanted to ask whether it is still being maintained or if it has been considered abandoned.

Are there any planned updates or a roadmap for the future?

If the project is no longer active, would you allow me to fork it in order to maintain it and continue its development?

Thank you in advance for your response!

**Comment by compwright** [comment]
@xalsie I'm the new maintainer. Rather than create a fork, submit PRs back to this project - help would definitely be welcome in these two areas:

1) Upgrading the tooling and tests to latest versions
2) Upgrading support for the latest node-redis client
